src/components/iteration-panel.js

```diff
@@ -2,30 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, Tooltip } from 'antd';
 import { useTranslation } from 'react-i18next';
-import { LeftOutlined, RightOutlined, UndoOutlined } from '@ant-design/icons';
+import { ChevronLeftIcon, ChevronLeftPipeIcon, ChevronRightIcon } from './icons/icons.js';
 
 function IterationPanel({ itemCount, selectedItemIndex, alwaysAllowPreviousClick, disabled, onNextClick, onPreviousClick, onResetClick }) {
   const { t } = useTranslation();
 
   return (
     <div className="IterationPanel">
-      <Button
-        shape="circle"
-        icon={<LeftOutlined />}
-        disabled={disabled || (!alwaysAllowPreviousClick && selectedItemIndex === 0)}
-        onClick={onPreviousClick}
-        />
       <Tooltip title={t('common:reset')} disabled={disabled}>
         <Button
           shape="circle"
-          icon={<UndoOutlined />}
+          icon={<ChevronLeftPipeIcon />}
```

Comment:
[This ticket ](https://educandu.atlassian.net/browse/EDU-1429) made me think that in this context the "reset" button could be confused with redo/replay, so I changed the icons to something hopefully clearer.